lib/data_update_scripts/20200519142908_re_index_feed_content_to_elasticsearch.rb

```diff
@@ -0,0 +1,28 @@
+module DataUpdateScripts
+  class ReIndexFeedContentToElasticsearch
+    def run
+      clear_existing_feed_documents
+
+      index_docs(Article.pluck(:id), "Article")
+      index_docs(PodcastEpisode.pluck(:id), "PodcastEpisode")
+      index_docs(Comment.pluck(:id), "Comment")
+    end
+
+    private
+
+    def clear_existing_feed_documents
+      # Clear out documents with incorrect IDs before reindex
+      Search::Client.delete_by_query(
+        index: Search::FeedContent::INDEX_ALIAS, body: { query: { match_all: {} } },
+      )
+    end
+
+    def index_docs(ids, doc_type)
+      ids.each do |id|
+        Search::IndexWorker.set(queue: :low_priority).perform_async(
```

Comment:
We now have a BulkIndex worker, we should use that here.  I would do batches of 100.